src/blocks/recommendations/block.json

```diff
@@ -0,0 +1,44 @@
+{
+	"$schema": "https://schemas.wp.org/wp/5.9/block.json",
+	"apiVersion": 2,
+	"name": "wp-parsely/recommendations",
+	"title": "Parse.ly Recommendations",
+	"category": "widgets",
+	"textdomain": "wp-parsely",
+	"supports": {
+		"html": false
+	},
+	"attributes": {
+		"boost": {
+			"type":"string",
+			"default": "views"
+		},
+		"imagestyle": {
+			"type": "string",
+			"default": "original"
+		},
+		"limit": {
+			"type": "number",
+			"default": 3
+		},
+		"showimages": {
+			"type": "boolean",
+			"default": true
+		},
+		"sort": {
+			"type":"string",
+			"default":"score"
+		},
+		"tag": {
+			"type":"string"
+		},
+		"title": {
+			"type":"string",
+			"default": "Related Content"
```

Comment:
Can this string be translated at some point?

Reply:
If I'm reading that right - that's only for top-level keys, and not an attribute's [default] value.

Reply:
Yes, I think you're correct. Update incoming.